core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -193,11 +193,11 @@ public String startWorkflow(String workflowId, String name, int version, Map<Str
 			}
 
 			WorkflowDef exists = metadata.get(name, version);
-			if (isOps != null && isOps) {
-				String validWorkflowsOps = config.getProperty("workflow.ops.auth.bypass", null);
+			String validWorkflowsOps = config.getProperty("workflow.ops.auth.bypass", null);
+			if (isOps != null && isOps && validWorkflowsOps != null) {
 				String[] strSplit = validWorkflowsOps.split(",");
 				Boolean found = false;
-				for (String workflowName :  Arrays.asList(strSplit)) {
+				for (String workflowName : Arrays.asList(strSplit)) {
```

Comment:
1. Need to break from the loop if the wf is found, otherwise it will fail if there are multiple wfs defined
2. use startsWith instead of contains